src/main/java/racing/view/ResultView.java

```diff
@@ -1,23 +1,47 @@
 package racing.view;
 
-import racing.RacingCars;
+import racing.domain.car.Car;
+import racing.domain.game.RacingGame;
+
+import java.util.List;
 
 public class ResultView {
+    private static final String DELIMITER = " : ";
+    private static final String COMMA = ", ";
+
     public static void resultMessage() {
         System.out.println("실행 결과");
     }
 
+    public static void printCarName(String carName) {
+        System.out.print(carName + DELIMITER);
+    }
+
     public static void printPosition(int carPosition) {
         for (int i = 0; i < carPosition; i++) {
             System.out.print("-");
         }
         System.out.println();
     }
 
-    public static void resultGame(RacingCars racingCars) {
-        for (int i = 0; i < racingCars.getCars().size(); i++) {
-            printPosition(racingCars.getCars().get(i).getPosition());
+    public static void printResult(List<Car> carPositions) {
+        for (int i = 0; i < carPositions.size(); i++) {
+            printCarName(carPositions.get(i).getCarName());
+            printPosition(carPositions.get(i).getPosition());
         }
         System.out.println();
     }
-}
\ No newline at end of file
+
+    public static void printWinner(RacingGame racingGame) {
+        List<String> winners = racingGame.getWinnersCarName();
+
+        StringBuilder stringBuilder = new StringBuilder();
+
+        winners.stream().forEach(value -> stringBuilder.append(value).append(COMMA));
+
+        stringBuilder.delete(stringBuilder.length() - 2, stringBuilder.length());
+        stringBuilder.append("가 최종 우승했습니다.");
+
+        System.out.println(stringBuilder);
```

Comment:
```suggestion
        List<String> winners = racingGame.getWinnersCarName();

        String winnerNames = winners.stream()
                .collect(Collectors.joining(COMMA));

        System.out.println(winnerNames + "가 최종 우승했습니다.");
```
위와 같이 사용할 수 있을것 같아요 !